src/Aperture/elliptical.jl

```diff
@@ -94,3 +94,57 @@ function mask(e::EllipticalAperture; method = :center)
     ny, nx = size(e)
     return elliptical_overlap(bounds..., nx, ny, e.a, e.b, e.theta, method = method)
 end
+
+
+#######################################################
+
+"""
+    EllipticalAnnulus(x, y, a, b, θ, factor)
+    EllipticalAnnulus([x, y], a, b, θ, factor)
+
+An elliptical annulus with semi-major axis `a`, semi-minor axis `b`, angle `θ` of inner ellipse ,scaling factor `factor` to generate the outer ellipse from inner ellipse.
+`a` and `b` must be ≥ 0, factor ≥ 1, `θ` is measured in degrees counter-clockwise the standard x-axis.
+
+# Examples
+```jldoctest
+julia> ap = EllipticalAnnulus(0, 0, 10, 5, 45, 2)
+EllipticalAnnulus(0, 0, a=10, b=5, θ=25°, factor=2)
+
+```
+"""
```

Comment:
Actually, I think the better way to parametrize this is
`EllipticalAnnulus(x, y, a_in, a_out, b_out, θ)`
and just calculate `b_in = a_in / a_out * b_out`.

This matches `photutils` so hopefully that removes a possible pain-point for future users.